examples/contracts/aggregator-consumer/scripts/consumerValidator.ts

```diff
@@ -0,0 +1,76 @@
+import { ethers, starknet, network } from 'hardhat'
+import { Contract, ContractFactory } from 'ethers'
+import { HttpNetworkConfig, StarknetContract, Account } from 'hardhat/types'
+import fs from 'fs'
+import dotenv from 'dotenv'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
+import { defaultProvider, ec } from 'starknet'
+import { loadContractSequencer } from '.'
+
+dotenv.config({ path: __dirname + '/.env' })
+const AGGREGATOR_NAME = 'Mock_Aggregator'
+const UPTIME_FEED_NAME = 'sequencer_uptime_feed'
+
+let Validator: Contract
+let MockStarknetMessaging: ContractFactory
+let mockStarknetMessaging: Contract
+let deployer: SignerWithAddress
+let eoaValidator: SignerWithAddress
+let networkUrl: string
+let account: Account
+
+async function main() {
+  // account = await starknet.getAccountFromAddress(
+  //   process.env.ACCOUNT_ADDRESS as string,
+  //   process.env.PRIVATE_KEY as string,
+  //   'OpenZeppelin',
+  // )
+
+  
+  // const MockUptimeFeedFactory = loadContractSequencer(UPTIME_FEED_NAME)
+  const MockUptimeFeedFactory = await starknet.getContractFactory('contracts/contracts/cairo/ocr2/SequencerUptimeFeed/sequencer_uptime_feed.cairo')
+  const MockUptimeFeedDeploy = await MockUptimeFeedFactory.deploy({})
+  // const MockUptimeFeedDeploy = await defaultProvider.deployContract({
+  //   contract: MockUptimeFeedFactory,
+  //   constructorCalldata: [0, account.address],
+  // })
```

Comment:
Please polish (remove unused code) and document the script purpose:

- inline as a quick one-liner
- + `examples/contracts/aggregator-consumer/README.md`